config/samples/dataplane_v1beta1_openstackdataplanenodeset.yaml

```diff
@@ -139,5 +130,3 @@ spec:
          # edpm firewall, change the allowed CIDR if needed
          edpm_sshd_configure_firewall: true
```

Comment:
can we change the default value in edpm-ansible and then drop this? I guess I don't see why the default is false in edpm-ansible, if we are just going to always set it to true in every sample.

Reply:
Yeah, that sounds like a good plan. I have no idea either, but my assumption was that someone did it for a reason. If neither of us know though, then I'll go propose changing it there.